app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command, origin, copy = command.split()
+    if origin != copy:
```

Comment:
Add, please, check that the cp input command is correct, as mentioned in the **checklist**